test/testvm.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, address=None, flavor=None, system=None, arch=None, verbose=Fa
         self.verbose = verbose
         self.flavor = flavor or DEFAULT_FLAVOR
 
-        conf_file = "%s.conf" % self.flavor
+        conf_file = "guest/%s.conf" % self.flavor
```

Comment:
Hubbot also reads these files.  I make a PR for hubbot so that it looks into both locations.  (Otherwise, hubbot might accidentally expire all images.)